packages/vuetify/src/components/VChip/VChip.sass

```diff
@@ -106,24 +110,16 @@
   .v-icon
     color: inherit
 
-  &:before
-    opacity: 0.22
-
 .v-chip--link
-  &:hover,
-  &:focus
-    &:before
-      opacity: 0.32
+  &::before
+    transition: opacity $primary-transition
+
+  &:focus::before
+    opacity: $chip-link-focus-opacity
```

Comment:
any reason for not using value from states here?

Reply:
Because the default state was almost unnoticeable so I used what was defined on the MWC chips.